azure-pipelines.yml

```diff
@@ -7,10 +7,56 @@ pool:
   vmImage: 'Ubuntu 16.04'
 
 steps:
-- script: echo Hello, world!
-  displayName: 'Run a one-line script'
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: 3.6
```

Comment:
upgrade to 3.7

Reply:
will do

Reply:
Side note 3.7.1 doesn't work. It's unsupported at this time

Reply:
unsupported by azure devops?